src/users.json

```diff
@@ -0,0 +1,10 @@
+[
+    {
+        "id": 1,
+        "name": "Chrispaix"
+    },
```

Comment:
Please kindly add other parameters like "first_name", "last_name" and "dob". Also replace the name with "username" which is the data you will get from the backend. "username" is also a requirement of the project.